conda_env/specs/__init__.py

```diff
@@ -2,13 +2,15 @@
 from .yaml_file import YamlFileSpec
 from .notebook import NotebookSpec
 from .requirements import RequirementsSpec
+from .jinja2_file import Jinja2Spec
 from ..exceptions import SpecNotFound
 
 all_specs = [
     BinstarSpec,
     NotebookSpec,
     YamlFileSpec,
-    RequirementsSpec
+    RequirementsSpec,
+    Jinja2Spec
```

Comment:
How is this supposed to be used from an user's point of view? I mean, if `Jinja2Spec` is the last one in this list, won't `YamlFileSpec` be end up being picked up instead every time?